migrations/sql/V2021.02.08_0852__ReportRun.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE orderlyweb_report_run
+(
+    key         TEXT      NOT NULL,
+    email       TEXT      NOT NULL,
+    date        TIMESTAMP NOT NULL,
+    report_name TEXT      NOT NULL,
```

Comment:
Yes, we can't make this a foreign key, can we, because the report won't be in the db the first time the report is run...

I think to be consistent with the fk in report_version, I'd be inclined to call this column 'report'...

Reply:
My instinct is always tablename_columnname but you're right - `report_name` is inconsistent and has a double meaning!